test/datasets/cmuarctic_test.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+from torchaudio.datasets import cmuarctic
+
+from ..common_utils import (
+    TempDirMixin,
+    TorchaudioTestCase,
+    get_whitenoise,
+    save_wav,
+    normalize_wav,
+)
+
+
+class TestCMUARCTIC(TempDirMixin, TorchaudioTestCase):
+    backend = "default"
+
+    root_dir = None
+    samples = []
+
+    @classmethod
+    def setUpClass(cls):
+        cls.root_dir = cls.get_base_temp_dir()
+        sample_rate = 16000
+        utterance = "This is a test utterance."
+
+        base_dir = os.path.join(cls.root_dir, "ARCTIC", "cmu_us_aew_arctic")
+        txt_dir = os.path.join(base_dir, "etc")
+        os.makedirs(txt_dir, exist_ok=True)
+        txt_file = os.path.join(txt_dir, "txt.done.data")
+        audio_dir = os.path.join(base_dir, "wav")
+        os.makedirs(audio_dir, exist_ok=True)
+
+        with open(txt_file, "w") as txt:
+            for c in ["a", "b"]:
+                for i in range(5):
+                    utterance_id = f"arctic_{c}{i:04d}"
+                    path = os.path.join(audio_dir, f"{utterance_id}.wav")
+                    data = get_whitenoise(
+                        sample_rate=sample_rate,
+                        duration=3,
+                        n_channels=1,
+                        dtype="int16",
+                        seed=i + ord(c),
```

Comment:
This does not produce unique values for all the generated data. Simply initialize the seed value outside of the most outer loop and increment it. See https://github.com/pytorch/audio/blob/master/test/datasets/gtzan_test.py#L27-L44

Reply:
Thanks @mthrok, sorry I overlooked this.